src/main/java/org/broadinstitute/hellbender/engine/spark/SparkCommandLineProgram.java

```diff
@@ -85,4 +93,36 @@ protected void afterPipeline(final JavaSparkContext ctx) {
     protected String getProgramName(){
         return programName == null ? getClass().getSimpleName() : programName;
     }
+
+    /**
+     * Returns the number of cores being used by each worker.
+     * In local mode it will return the total number of cores allocated to the process.
+     */
+    protected int getExecutorCores() {
+        int cores;
+        SchedulerBackend schedulerBackend = ctx.sc().schedulerBackend();
```

Comment:
The schedulerBackend method is undocumented as of Spark 1.4.  Maybe not wise to rely on it?